bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/dto/RuleDTO.java

```diff
@@ -16,6 +16,7 @@
 import java.util.Map;
 import java.util.Set;
 
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
When adding null annotations, we typically add `@NonNullByDefault` to the class and then add `@Nullable` whereever appropriate.

Reply:
Ok, if it is done similarly on other DTOs, let's keep it; I get your point.